client/components/.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "settings": {
+    "ecmascript": 6,
+    "jsx": true
+  },
```

Comment:
With the eslint-plugin-react plugin, the readme says that you should incluce `"jsx": true` in the settings, so I've left that portion there.